Tests/ArcGISToolkitTests/LocationButtonTests.swift

```diff
@@ -0,0 +1,143 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Testing
+import ArcGIS
+@testable import ArcGISToolkit
+
+@Suite("LocationButton Tests")
+struct LocationButtonTests {
+    @Test
+    @MainActor
+    func testInit() {
+        let locationDisplay = LocationDisplay(dataSource: MockLocationDataSource())
+        let button = LocationButton(locationDisplay: locationDisplay)
+        
+        #expect(button.locationDisplay === locationDisplay)
+        #expect(button.autoPanOptions == [.recenter, .compassNavigation, .navigation, .off])
+        #expect(button.status == .stopped)
+        #expect(button.autoPanMode == .off)
+        #expect(!button.buttonIsDisabled)
```

Comment:
You'll find that you get better failure messages when you always use an equal sign with a value on the right side.

```suggestion
        #expect(!button.buttonIsDisabled)
```

Before | <img width="1235" height="174" alt="image" src="https://github.com/user-attachments/assets/711a666e-070f-4593-b093-bebd4de6c47a" />
--- | ---
After | <img width="1232" height="133" alt="image" src="https://github.com/user-attachments/assets/0f9910d1-bebc-44d3-ab8b-89ef98dc145c" />